demisto_sdk/main.py

```diff
@@ -1,11 +1,292 @@
+    help='Is CircleCi or not')
+@click.option(
+    '-wl', '--whitelist', default='./Tests/secrets_white_list.json', show_default=True,
+    help='Full path to whitelist file, file name should be "secrets_white_list.json"')
+@pass_config
+def secrets(config, **kwargs):
+    sys.path.append(config.configuration.env_dir)
+    validator = SecretsValidator(configuration=config.configuration, is_circle=str2bool(kwargs['circle']),
+                                 white_list_path=kwargs['whitelist'])
+    return validator.run()
+
+
+# ====================== lint ====================== #
+@main.command(name="lint",
+              short_help="Run lintings (flake8, mypy, pylint, bandit) and pytest. pylint and pytest will run within the"
+                         "docker image of an integration/script. Meant to be used with integrations/scripts that use "
+                         "the folder (package) structure. Will lookup up what docker image to use and will setup the "
+                         "dev dependencies and file in the target folder. ")
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    "-d", "--dir", help="Specify directory of integration/script", required=True)
+@click.option(
+    "--no-pylint", is_flag=True, help="Do NOT run pylint linter")
+@click.option(
+    "--no-mypy", is_flag=True, help="Do NOT run mypy static type checking")
+@click.option(
+    "--no-flake8", is_flag=True, help="Do NOT run flake8 linter")
+@click.option(
+    "--no-bandit", is_flag=True, help="Do NOT run bandit linter")
+@click.option(
+    "--no-test", is_flag=True, help="Do NOT test (skip pytest)")
+@click.option(
+    "-r", "--root", is_flag=True, help="Run pytest container with root user")
+@click.option(
+    "-k", "--keep-container", is_flag=True, help="Keep the test container")
+@click.option(
+    "-v", "--verbose", is_flag=True, help="Verbose output")
+@click.option(
+    "--cpu-num",
+    help="Number of CPUs to run pytest on (can set to `auto` for automatic detection of the number of CPUs.)",
+    default=0)
+@pass_config
+def lint(config, dir, **kwargs):
+    linter = Linter(configuration=config.configuration, project_dir=dir, **kwargs)
+    return linter.run_dev_packages()
+
+
+# ====================== run-playbook ====================== #
+@main.command(name="run-playbook",
+              short_help="Run a playbook in Demisto.")
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    '--url', '-u',
+    help='URL to a Demisto instance.',
+    required=True
+)
+@click.option(
+    '--playbook_id', '-p',
+    help="The playbook ID to run.",
+    required=True
+)
+@click.option(
+    '--wait', '-w',
+    type=click.Choice(["True", "False"]),
+    default="True",
+    show_default=True,
+    help="Wait until the playbook run is finished and get a response."
+)
+@click.option(
+    '--api', '-a',
+    required=True,
+    help="API key to connect with"
+)
```

Comment:
remove this, not secure as we talked in person

Reply:
Done